system-design/app-interoperability/local-registries.md

```diff
@@ -0,0 +1,85 @@
+# Local Registries
+
+This section investigates options for configuring the usage of local Docker (or Helm Chart) registries. The goal of configuring such local registries is to avoid the reliance on public, Internet-accessible registries. The reasons for not using such public registries are mainly twofold: (1) publicly hosted Docker images or Helm charts could become unavailable at some point, as the owner decides to take the Docker images or Helm charts off the public registry, or (2) Internet connectivity may not be available to the device and hence public registries are not reachable.
+

```

Comment:
I think a third point would be that end-users want to host their own registries so they can do security scans and validate the packages

Reply:
yes, I added this point to the text and you find it in the latest commit.